x/ccv/provider/keeper/relay.go

```diff
@@ -300,24 +317,63 @@ func (k Keeper) HandleSlashPacket(ctx sdk.Context, chainID string, data ccv.Slas
 // EndBlockCIS contains the EndBlock logic needed for
 // the Consumer Chain Removal sub-protocol
 func (k Keeper) EndBlockCCR(ctx sdk.Context) {
```

Comment:
Is this method tested? We should prob create an issue if not, since there's a good amount of logic here now

Reply:
It's tested in the e2e tests. There is no UT though.

Reply:
Cool that should be fine 👍